docs/document/content/others/faq/_index.cn.md

```diff
@@ -194,26 +269,13 @@ git config --global core.longpaths true
 https://docs.microsoft.com/zh-cn/windows/desktop/FileIO/naming-a-file
 https://ourcodeworld.com/articles/read/109/how-to-solve-filename-too-long-error-in-git-powershell-and-github-application-for-windows
 
-## 14. Windows环境下，运行ShardingSphere-Proxy，找不到或无法加载主类 org.apache.shardingsphere.proxy.Bootstrap，如何解决？
-
-回答：
-
-某些解压缩工具在解压ShardingSphere-Proxy二进制包时可能将文件名截断，导致找不到某些类。
-
-解决方案：
-
-打开cmd.exe并执行下面的命令：
-```
-tar zxvf apache-shardingsphere-${RELEASE.VERSION}-shardingsphere-proxy-bin.tar.gz
-```
-
-## 15. Type is required 异常的解决方法?
+## 22. [其他] Type is required 异常的解决方法?
 
 回答：
 
 ShardingSphere中很多功能实现类的加载方式是通过[SPI](https://shardingsphere.apache.org/document/current/cn/features/pluggable-architecture/)注入的方式完成的，如分布式主键，注册中心等；这些功能通过配置中type类型来寻找对应的SPI实现，因此必须在配置文件中指定类型。
 
-## 16. 为什么我实现了`KeyGenerateAlgorithm`接口，也配置了Type，但是自定义的分布式主键依然不生效？
+## 23. [其他] 为什么我实现了`KeyGenerateAlgorithm`接口，也配置了Type，但是自定义的分布式主键依然不生效？
 
```

Comment:
Should the content related to `KeyGenerateAlgorithm` use the `[sharding]` tag?

I found that this interface is in the `shardingsphere-sharding` module.

Reply:
> Hi @yx9o ,
> Should the content related to `KeyGenerateAlgorithm` use the `[sharding]` tag?
> I found that this interface is in the `shardingsphere-sharding` module.

Ok, is there anything else incorrect? Let me modify it together.